contents/docs/alerts/index.mdx

```diff
@@ -0,0 +1,128 @@
+
+8. (Optional) For relative alerts ('increases by'/'decreases by'), you can set a percentage threshold. It compares the percentage change in the series value to the thresholds you've set.
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/percentage_threshold_1940ead9e6.png" 
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/percentage_threshold_1940ead9e6.png"
+    alt="percentage threshold" 
+    classes="rounded"
+/>
+
+9. Select how often you want the alert to be checked (e.g. every hour, every day, every week, every month)
+
+10. Pick who all should be notified when the alert is triggered. The alert will send an email to everyone in this list when the alert is triggered.
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/filled_alert_627158e6bb.png" 
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/filled_alert_627158e6bb.png"
+    alt="filled alert" 
+    classes="rounded"
+/>
+
+11. Click 'Create alert', and you're done! 
+
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/insight_alerts_5k_daef47ade1.png" 
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/insight_alerts_5k_daef47ade1.png"
+    alt="insight alerts" 
+    classes="rounded"
+/>
+
+You can now view all the alerts set on the insight. 
+To view all the alerts set, you can click on ['Product Analytics'](/product-analytics) in the sidebar and then click on the 'Alerts' tab.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/all_alerts_d92386e45a.png" 
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/all_alerts_d92386e45a.png"
+    alt="all alerts" 
+    classes="rounded"
+/>
+
+
+## Relative alerts
+Relative alerts check for change in the value of an insight.
+For example, if a value increases by 5% in a week.
+To create a relative alert, you just need to change the 'has value' option to 'increases by' or 'decreases by'.
+This will then also enable you to set a percentage threshold.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/relative_alert_picker_fa04215397.png" 
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/relative_alert_picker_fa04215397.png"
+    alt="relative alert picker" 
+    classes="rounded"
+/>
+
+
+### Thresholds
+We support both **absolute value** thresholds (insight value more than or less than a certain number) or **percentage** thresholds (insight value changed by a certain percentage).
+Percentage thresholds are only available for relative alerts (as you need to compare two values to figure out percentage change)
+
+
+## Notification
+Currently alerts only support email notifications. We will add more notification options like webhooks and slack soon.
+
+
+## Alerts on breakdowns
+When you set an alert on a trend with a breakdown. 
```

Comment:
I think this is a partial sentence. You can probably replace the period with a comma, and then pull the next line up to complete the sentence.